### ISSUE TITLE
core.pause_torrent fails on 1.3.15

### ISSUE THREAD

**Issue by l3uddz** [gdm85/go-libdeluge#30]
See below log message:

```
[2020-04-18T19:22:25+02:00]  INFO manage        : Hard removing: "xxxxxxxxxxxxxxxxx.mkv" - 3.4 GiB - 0.00 GB
[2020-04-18T19:22:25+02:00]  INFO manage        : Ratio: 1.801 / Seed days: 1.256 / Seeds: 76 / Label: sonarrimported / Tracker: XXXXX / Tracker Status: "XXXXX: Announce OK"
[2020-04-18T19:22:25+02:00] FATAL manage        : Failed removing torrent: {Hash:da8a43ece27548933d533a32af786bb81a4079e7 Name:xxxxxxxxxxxxxx.mkv Path: TotalBytes:3675936559 DownloadedBytes:3675936559 State:Seeding Files:[xxxxxxxxxxxxxx.mkv] Downloaded:true Seeding:true Ratio:1.8010296 AddedSeconds:110685 AddedHours:30.745832 AddedDays:1.2810763 SeedingSeconds:108522 SeedingHours:30.144999 SeedingDays:1.2560416 Label:sonarrimported Seeds:76 Peers:0 FreeSpaceGB:0x925c40 FreeSpaceSet:true TrackerName:XXXXX TrackerStatus:XXXXX: Announce OK} error=RPC error KeyError('d')
Traceback:   File "/usr/lib/python2.7/dist-packages/deluge/core/rpcserver.py", line 301, in dispatch
    ret = self.factory.methods[method](*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/deluge/core/core.py", line 403, in pause_torrent
    if not self.torrentmanager[torrent_id].pause():
  File "/usr/lib/python2.7/dist-packages/deluge/core/torrentmanager.py", line 287, in __getitem__
    return self.torrents[torrent_id]

failed pausing torrent: "da8a43ece27548933d533a32af786bb81a4079e7"
github.com/l3uddz/tqm/client.(*Deluge).RemoveTorrent
        github.com/l3uddz/tqm/client/deluge.go:211
github.com/l3uddz/tqm/cmd.removeEligibleTorrents
        github.com/l3uddz/tqm/cmd/helpers.go:77
github.com/l3uddz/tqm/cmd.glob..func1
        github.com/l3uddz/tqm/cmd/manage.go:97
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.0.0/command.go:846
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.0.0/command.go:950
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.0.0/command.go:887
github.com/l3uddz/tqm/cmd.Execute
        github.com/l3uddz/tqm/cmd/root.go:41
main.main
        github.com/l3uddz/tqm/main.go:8
runtime.main
        runtime/proc.go:203
runtime.goexit
        runtime/asm_amd64.s:1373
```

From the above RPC error, it would appear the full hash is not being sent.

This is on v0.5.2.

I dont have a buffer example but will try to get one if that will help.

**Comment by gdm85** [comment]
Thanks for your report; we are discussing a fix in #31. This bug was not caught because the integration test is missing, we will add it as well so that both v1 and v2 methods are always tested.

> I dont have a buffer example but will try to get one if that will help.

It has become easier with the GitHub Actions here: once the integration test is added the buffer can be copied from the console output.

**Comment by l3uddz** [comment]
No problem!

Thanks for the awesome package and the additional information r.e. integration tests (will be handy for future PR's i send here)!
